### PR TITLE
refs ST-3542[*]: Add mode differentiation

### PR DESCRIPTION
**Pull request by ASenart**

### CODE REVIEW

**Comment by trungkienbkhn** [comment]
@ASenart , hello. Which version of CM2 storage can verify your updated logic, and to which ticket is it related ?

**Comment by ASenart** [comment]
Hi, this is not linked to a new version of CM2, it's more related to this comment : https://systran.atlassian.net/browse/ST-3542?focusedCommentId=27175

**Comment by trungkienbkhn** [comment]
In your comment, feature 'es1' is filled in as 'yes' for lite mode. But it's disable in your PR logic. Is there any confusion here?
Besides, I think you should add refs ticket to your PR, it will be easier to check the logic involved.

**Comment by trungkienbkhn** [comment]
```python
if not isAdvanced:
    import_options["cleanFormatting"] = True
    import_options["removeDuplicates"] = True
else:
   features = 'es1'
```
But in the above logic (if I understand correctly), feature 'es1' is only set in mode advanced, right ? If 'es1' is default behavior in CM2, this logic above can be misleading.

**Comment by ASenart** [comment]
Does that make it clearer?

File: systran_storages/storages/corpus.py [hunk 1]
```diff
@@ -435,23 +435,34 @@ def push_file(self, local_path, remote_path, lp=None):
             data = f.read()
             format_path = self._get_format_from_local_path(local_path)
             import_options = {
-                "cleanFormatting": True,
-                "removeDuplicates": True,
                 "tidyLocale": True
             }
+            features = None
```
Comment (trungkienbkhn) [inline systran_storages/storages/corpus.py:440]: should remove unnecessary logic

**Comment by trungkienbkhn** [comment]
Yes, it's more clearer. But it still lacks a logic, isAdvanced param has not been added to the function in [storages/systran_storages/storage.py](https://github.com/SYSTRAN/storages/blob/master/systran_storages/storage.py#L177)
And I think isAdvanced should be renamed to is_advanced according to python snake_case standard

File: systran_storages/storages/generic.py [hunk 2]
```diff
@@ -185,12 +185,12 @@ def stream_corpus_manager(self, remote_id, remote_format, buffer_size=1024):
         raise NotImplementedError()
 
     @abc.abstractmethod
-    def push_file(self, local_path, remote_path, lp=None):
+    def push_file(self, local_path, remote_path, lp=None, is_advanced= False):
```
Comment (trungkienbkhn) [inline systran_storages/storages/generic.py:188]: Minor syntax warning: unexpected spaces

**Comment by trungkienbkhn** [comment]
@ASenart , your PR looks good to me now. Could you fix the small syntax, and then I will merge it? Thanks.

**Pull request merged**